.github/workflows/performance-benchmarks-bayesmark.yml

```diff
@@ -90,22 +91,32 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@master
+    - name: Checkout
+      uses: actions/checkout@master
 
-    - uses: actions/download-artifact@v2
+    - name: Download partial reports
+      uses: actions/download-artifact@v2
       with:
         name: partial-reports
         path: partial
     
+    - name: Install Python libraries
+      run: |
+        python -m pip install --upgrade pip
+        pip install --progress-bar off -U setuptools wheel
+        pip install --progress-bar off numpy scipy pandas
```

Comment:
How about merging this install section to that of Line 53?

Reply:
This is install section of `report` job, while on L53 we have installs for `benchmarks` job - both have separate environments and installs. AFAIK there is no way to define common job steps between jobs (like GitLab [.before_script](https://docs.gitlab.com/ee/ci/yaml/#before_script)), but I could be wrong here.

Reply:
Thanks for the explanation. You're right.